README.md

```diff
@@ -38,11 +38,12 @@ A Ruby client for the Slack [Web](https://api.slack.com/web), [RealTime Messagin
       - [Rate Limiting](#rate-limiting)
       - [Other Errors](#other-errors)
   - [RealTime Client](#realtime-client)
-    - [Slack::RealTime::Stores::Store](#slackrealtimestoresstore)
-    - [Slack::RealTime::Stores::Starter](#slackrealtimestoresstarter)
-  - [Configuring Slack::RealTime::Client](#configuring-slackrealtimeclient)
-    - [Caveats](#caveats)
-      - [websocket_ping](#websocket_ping)
+    - [Configuring Slack::RealTime::Client](#configuring-slackrealtimeclient)
+      - [Caveats](#caveats)
+        - [websocket_ping](#websocket_ping)
+    - [RealTime Store](#realtime-store)
+      - [Slack::RealTime::Stores::Starter](#slackrealtimestoresstarter)
+      - [Slack::RealTime::Stores::Store](#slackrealtimestoresstore)
```

Comment:
I could see renaming these to something like `Light` and `Full` to be more clear, since `Store` along isn't very clear, but I don't know if there's much gain.